templates/level-struktur/level-struktur.css

```diff
@@ -6,4 +6,12 @@ body.level-struktur {
 body.level-struktur h1 > strong{
  font-size: var(--heading-font-size-xxl);
  font-weight: var(--font-weight-bold);
```

Comment:
the font-weight isn't needed here, it should inherit from styles.css (which is 900).